libs/barista-components/formatters/src/formatters-module.ts

```diff
@@ -36,7 +36,7 @@ const FORMATTERS = [
   DtPercent,
   DtRate,
   DtDateRange,
-  DtTime,
```

Comment:
I guess we cannot remove it here, otherwise it would be a breaking change. 
Add the DtTime back in and add a deprecation and breaking change comment.